redisson/src/main/java/org/redisson/Redisson.java

```diff
@@ -651,7 +652,12 @@ public RFencedLock getFencedLock(CommonOptions options) {
     public RLock getMultiLock(RLock... locks) {
         return new RedissonMultiLock(locks);
     }
-    
+
+    @Override
+    public RLock getFasterMultiLock(String group, Collection<String> values) {
```

Comment:
Should be renamed to getMultiLock() and `group` to `name`. 

As for values can you make it an Object so not only Strings can be allowed?

Reply:
Method has been renamed to getMultiLock.
The param `values` just like name of RedissonLock,  it is actually used as the field name of hash type， and i will concat it with ':lock_count' as lock count field name. it will be not easy to concat if it's object.
Maybe it  should named `keys`?

Reply:
> Should be renamed to getMultiLock() and `group` to `name`.
> 
> As for values can you make it an Object so not only Strings can be allowed?

Method has been renamed to getMultiLock.
The param `values` just like name of RedissonLock,  it is actually used as the field name of hash type， and i will concat it with ':lock_count' as lock count field name. it will be not easy to concat if it's object.
Maybe it  should named `keys`?

Reply:
> Should be renamed to getMultiLock() and `group` to `name`.
> 
> As for values can you make it an Object so not only Strings can be allowed?

Or do you have any suggestions to concat string and object？

Reply:
it should be a hash value

Reply:
> it should be a hash value
Sorry.
I don't understand why it shoule be hash value .
RedissonFasterMultiLock just like RedissonLock, and support multi Lock.
In RedissonLock,  the lock value is String, and it is hash key, so you can save other data in this hash data.
If it is a hash value , and object, how can i know it is locked or not?  get and iterate all data ? The efficiency is too low.
And there are other problems like how can i save lock count and expire time?

Reply:
> Sorry, I mean this hash:
> 
> ```java
>     protected String keyHash(Object key) {
>         ByteBuf objectState = encode(key);
>         try {
>             return Hash.hash128toBase64(objectState);
>         } finally {
>             objectState.release();
>         }
>     }
> ```

Ha ha ha，my apology.
Please  check.